src/algorithms/clustering/gmm.py

```diff
@@ -132,6 +136,17 @@ def log_likelihood(self):
             result += np.log(row_result)
         return result
 
+    def update_metrics(self, labels):
+        self.metrics_info = {}
+        d_index = dunn_score(self.df, labels)
+        db_index = davies_bouldin_score(self.df, labels)
+        s_index = silhouette_score(self.df, labels)
+        self.metrics_info["Dunn index (higher = better)"] = round(d_index, 3)
```

Comment:
ideally this "(higher = better)" type info should be in a tooltip. we would save a tuple of label, tooltip info here. but this way is good enough for now, so it can stay as it is for me.

one thing that bothers me a little is that because this info is so long it makes the whole left column in the clustering results section wider and it makes the whole window wider too. however that's an issue in many places and not only this one. don't know if there is a way to stop widgets from changing window dimensions, but I could look into this if we decide that it's worth fixing now